### PR TITLE
insp

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-12-12-inspiration.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+status: publish
+categories: [Writing]
+image:
+image2:
+author: Valentino Urbano
+---
+
+If you ask someone who doesn't write often either of the following questions:
+
+- If you had to write something where do you find inspiration on?
```
Comment (MsSlacker) [inline _posts/2019-12-12-inspiration.md]: Where would you find the inspiration + and no "on"

File: _posts/2019-12-12-inspiration.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- What would you write about?
+
+They would probably say that they find inspiration on important event through their life, or that they find it in their daily routine or again that they would write about what they have already knowledge of.
```
Comment (MsSlacker) [inline _posts/2019-12-12-inspiration.md]: On - > from

Comment (MsSlacker) [reply _posts/2019-12-12-inspiration.md]: already have

Comment (MsSlacker) [reply _posts/2019-12-12-inspiration.md]: the knowledge

File: _posts/2019-12-12-inspiration.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+It might seem hard to find topics to write about at first, but with time you will find yourself with lot of ideas.
```
Comment (MsSlacker) [inline _posts/2019-12-12-inspiration.md]: Lot - > lots

File: _posts/2019-12-12-inspiration.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+The problem will change and become how to capture them effectively and how to know which ones to write about and which ones to discard.
+
+Some ideas are plainly bad, but for those kind of ideas you wouldn't even consider writing it down in your idea list so we are going to ignore those. For all the rest you can't know if an idea is good until you start experimenting with it. After writing a few short sentences it will become obvious if it's an idea that it worth pursuing or not.
```
Comment (MsSlacker) [inline _posts/2019-12-12-inspiration.md]: Bad ideas can still be good with proper execution

/R/ATBGE

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-12-12-inspiration.md (hunk 4 above)
Comment (valeIT) [reply _posts/2019-12-12-inspiration.md]: Not sure I wanted to see that.. xD but good point

Comment (valeIT) [reply _posts/2019-12-12-inspiration.md]: Added footnote

**Review by MsSlacker — APPROVED** [review]
Lol

**Pull request merged**